### DISCUSSION TITLE
Is it possible to play PokeMMO on this system?

### DISCUSSION THREAD

**Discussion by JEFEXIS** [OnionUI/Onion#1402] [Q&A]
Can the system developers create a way to play PokeMMO on this system?

**Comment by XK9274** [comment]
I've tried this. 

Java is a little too CPU heavy, maybe a bit optimistic to think it'll run on 128mb RAM, too. 

The JDK is about 400mb, so not easily distributed.

It needs a compositor, which i have working to a degree (weston)

Someone on reddit took over from where i got up to but haven't heard anything since. If anyone else wants to have a go then contact me & and i'll drop you the files i have.
